components/library/tdb/Makefile

```diff
@@ -11,40 +11,50 @@
 
 #
 # Copyright 2017 Alexander Pyhalov
+# Copyright (c) 2022, Friedrich Kink
 #
 
+BUILD_BITS =                    64
+BUILD_STYLE =                   waf
+
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= tdb
-COMPONENT_VERSION= 1.3.12
+COMPONENT_VERSION= 1.4.7
 COMPONENT_REVISION= 1
 COMPONENT_SUMMARY= Trivial Database Library
```

Comment:
Remove COMPONENT_REVISION

Reply:
oh sorry I have simply overseen this one.